app/bundles/ReportBundle/Entity/Report.php

```diff
@@ -127,6 +132,11 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->columnName('aggregators')
             ->nullable()
             ->build();
+
+        $builder->createField('settings', 'array')
```

Comment:
Let's not use `array` type for new columns since it's slow, hard to read by humans, it creates longer strings and it often fails during decoding. Let's use [json-array](http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#json-array) instead.

The migration will have to be updated too.